apps/settings/js/panels/developer_service_workers/developer_service_workers.js

```diff
@@ -0,0 +1,257 @@
+/* -*- Mode: Java; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
+/* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global LazyLoader */
```

Comment:
These two lines are not required if we use AMD require for loading uuid. Please check my comments below.